src/envoy/auth/jwt_test.cc

```diff
@@ -329,90 +331,129 @@ TEST_F(JwtTest, InvalidSignature) {
   auto invalid_jwt = kJwt;
   invalid_jwt[kJwt.length() - 1] = kJwt[kJwt.length() - 1] != 'a' ? 'a' : 'b';
```

Comment:
Modifying the last character may make the signature part an invalid base64url input